README.md

```diff
@@ -24,34 +24,39 @@ To run the bootstrap task, use the bolt command, specifying the PE master from w
 
 #### Example: Basic usage
 
-On the command line, run `bolt task run bootstrap master=<master's fqdn> --nodes x,y,z --modules /path/to/modules`
+On the command line:
+* For Linux,   run `bolt task run bootstrap::linux   master=<master's fqdn> --nodes x,y,z --modules /path/to/modules`
+* For Windows, run `bolt task run bootstrap::windows master=<master's fqdn> --nodes x,y,z --modules /path/to/modules`
+For all advanced examples below, simply replace `bootstrap::linux` by `bootstrap::windows` to perform the action on Windows.
```

Comment:
Needs a preceding newline to display properly.

Reply:
There are no examples for running this on a Windows host i.e. `--modules /path/to/modules` is not an absolute path on Windows.

Reply:
`For Windows ...` is ambiguous.  That's to install a agent on Windows node, not using this task on a Windows host.